runtime/src/main/resources/META-INF/quarkus-extension.yaml

```
@@ -1,9 +1,12 @@
-name: Jdbc Singlestore
-#description: Do something useful.
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "JDBC Driver - Singlestore"
 metadata:
-#  keywords:
-#    - jdbc-singlestore
-#  guide: https://quarkiverse.github.io/quarkiverse-docs/jdbc-singlestore/dev/ # To create and publish this guide, see https://github.com/quarkiverse/quarkiverse/wiki#documenting-your-extension
-#  categories:
-#    - "miscellaneous"
-#  status: "preview"
+  keywords:
+    - "jdbc-singlestore"
+    - "jdbc"
+    - "singlestore"
+  guide: "https://quarkus.io/guides/datasource"
+  categories:
+    - "data"
+  status: "stable"
```

I'd recommend starting with a preview status since it's pretty recent
```suggestion
  status: "preview"
```